helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -11,6 +11,11 @@ extensions:
     directory: {{ .Values.logsCollection.checkpointPath }}
   {{- end }}
 
+  {{- if or .Values.splunkPlatform.persistentQueueEnabled.logs .Values.splunkPlatform.persistentQueueEnabled.metrics }}
```

Comment:
you defined the `"splunk-otel-collector.persistentQueueEnabled"` template for this purpose